### PR TITLE
Add Doctrine DHMiddleware by default

### PR DESCRIPTION
**Pull request by dmitryuk**
Closes https://github.com/DamienHarper/auditor/issues/113

### CODE REVIEW

**Comment by dmitryuk** [comment]
@DamienHarper I need a new release of https://github.com/DamienHarper/auditor to make test work

**Comment by DamienHarper** [comment]
auditor `2.2.0` is released

File: tests/App/Command/CreatePostCommand.php [hunk 1]
```diff
@@ -15,15 +15,20 @@
 
 class CreatePostCommand extends Command
 {
-    protected static $defaultName = 'app:post:create';
     private DoctrineProvider $doctrineProvider;
 
     public function __construct(DoctrineProvider $doctrineProvider)
     {
-        parent::__construct(self::$defaultName);
+        parent::__construct();
```
Comment (dmitryuk) [inline tests/App/Command/CreatePostCommand.php:22]: Fix deprecations

File: tests/App/config/packages/sf6/framework.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 framework:
     secret: '%env(APP_SECRET)%'
     test: true
+    http_method_override: false
```
Comment (dmitryuk) [inline tests/App/config/packages/sf6/framework.yaml:4]: Fix deprecation

**Comment by dmitryuk** [comment]
Tests anyway will throw `User Deprecated: The "Symfony\Bridge\Doctrine\Logger\DbalLogger" class implements "Doctrine\DBAL\Logging\SQLLogger" that is deprecated Use {@see \Doctrine\DBAL\Logging\Middleware} or implement {@see \Doctrine\DBAL\Driver\Middleware} instead.` because Doctrine anyway defines the `doctrine.dbal.logger` service and Symfony includes the class.

But now the error not caused by `auditor` package and must be resolved outside.

image
image


PR is ready for review and merge

File: tests/DependencyInjection/Compiler/DoctrineMiddlewareCompilerPassTest.php [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+ * @internal
+ *
+ * @small
+ */
+final class DoctrineMiddlewareCompilerPassTest extends AbstractCompilerPassTestCase
+{
+    public function testCompilerPass(): void
+    {
+        if (!interface_exists(Middleware::class) || !class_exists(DHMiddleware::class)) {
+            self::markTestSkipped('DHMiddleware doesn\'t supported');
```
Comment (DamienHarper) [inline tests/DependencyInjection/Compiler/DoctrineMiddlewareCompilerPassTest.php:32]: Small typo here: **isn't** instead of **doesn't**

**Review by DamienHarper — CHANGES_REQUESTED** [review]

**Comment by DamienHarper** [comment]
@dmitryuk Thanks a lot for your hard work

File: tests/DependencyInjection/Compiler/DoctrineMiddlewareCompilerPassTest.php (hunk 3 above)
Comment (dmitryuk) [reply tests/DependencyInjection/Compiler/DoctrineMiddlewareCompilerPassTest.php]: done

**Review by DamienHarper — APPROVED** [review]

**Comment by dmitryuk** [comment]
@DamienHarper can you make a release?

**Comment by DamienHarper** [comment]
@dmitryuk done

**Pull request merged**